src/Input/TextField.js

```diff
@@ -251,6 +260,11 @@ class TextField extends React.Component {
 
     var inputId = this.props.id || UniqueId.generate();
 
+    var searchButtonElement = this.props.searchButton ? 
+      (<IconButton onClick={this._handleInputFocus} key='searchGo'>
+          <SearchIcon />
```

Comment:
Do not need to add Search Icon here since Text Field should just display text input